### PR TITLE
Pass exposed schemas to lint query

### PR DESCRIPTION
**Pull request by saltcod**
Pass a string of exposed schemas to the lint query

screenshot-2024-05-06-at-16 52 18

### CODE REVIEW

File: apps/studio/data/lint/lint-query.ts [hunk 1]
```diff
@@ -841,13 +844,19 @@ export type ProjectLintsError = unknown
 
 export const useProjectLintsQuery = <TData = ProjectLintsData>(
   { projectRef, connectionString }: ProjectLintsVariables,
```
Comment (charislam) [inline apps/studio/data/lint/lint-query.ts:846]: very minor nit: `ProjectLintsVariables` doesn't omit `exposedSchemas` here, so it's possible to pass it in when calling the hook from elsewhere (gets ignored, but could be confusing DX)

**Review by charislam — APPROVED** [review]

File: apps/studio/data/lint/lint-query.ts (hunk 1 above)
Comment (saltcod) [reply apps/studio/data/lint/lint-query.ts]: aah great catch! I updated this — did I do it right @charislam ?

Comment (charislam) [reply apps/studio/data/lint/lint-query.ts]: yep works now! 👍🏼

Comment (saltcod) [reply apps/studio/data/lint/lint-query.ts]: nice, great catch!

**Pull request merged**